.travis.yml

```diff
@@ -24,6 +24,8 @@ language: python
 matrix:
   include:
     - python: "3.4"
+      env: TEST_SUITE=check-capitalization
+    - python: "3.4"
```

Comment:
Hmm, we need to install dependences for this, which takes a few minutes; I would probably add this to `backend` since that's where our linters are?